echopype/echodata/simrad.py

```diff
@@ -118,11 +120,12 @@ def _retrieve_correct_beam_group_EK80(
     power_ed_group = None
     complex_ed_group = None
 
+    transmit_type_has_BB = not np.all(echodata["Sonar/Beam_group1"]["transmit_type"] == "CW")
     if waveform_mode == "BB":
         # check BB waveform_mode, BB must always have complex data, can have 2 beam groups
-        # when echodata contains CW power and BB complex samples, and frequency_start
+        # when echodata contains CW power and BB complex samples, and transmit_frequency_start
         # variable in Beam_group1
```

Comment:
```suggestion
        # when echodata contains CW power and BB complex samples.
```
I think we can safely remove the rest of the sentence since now `transmit_frequency_start` would always exist.

Reply:
Right, thanks. Will do.